metrics-rusage.opam

```
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer:   "thomas@gazagnaire.org"
+authors:      ["Thomas Gazagnaire"]
+license:      "ISC"
+homepage:     "https://github.com/mirage/metrics"
+bug-reports:  "https://github.com/mirage/metrics/issues"
+dev-repo:     "git+https://github.com/mirage/metrics.git"
+doc:          "https://mirage.github.io/metrics/"
+
+build: [
+  ["dune" "subst"] {pinned}
```

```suggestion
  ["dune" "subst"] {dev}
```

See https://github.com/ocaml/opam-repository/pull/18732

---

fixed, also in all the other opam files.